pkg/controllers/default_resources.go

```diff
@@ -479,3 +479,20 @@ func ScaffoldAzureCloudCredential(hyd *hypdeployment.HypershiftDeployment, creds
 		},
 	}
 }
+
+func scaffoldSSHCredential(hyd *hypdeployment.HypershiftDeployment, sshPublicKey, sshPrivateKey []byte) *corev1.Secret {
+	return &corev1.Secret{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "Secret",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      hyd.Name + "-ssh-key",
+			Namespace: helper.GetHostingNamespace(hyd),
+		},
+		Data: map[string][]byte{
```

Comment:
Is this what the hypershift CLI does, I guess to support the bastion, as the private key in theory is for accessing the cluster, and the nodes should only be populated with the public key.